### DISCUSSION TITLE
'output' folder structure is not same as 'watch'

### DISCUSSION THREAD

**Discussion by techpara** [jlesage/docker-handbrake#378] [Q&A] [answered]
If I add multiple folders inside 'watch' but all converted files from all folder saved only in 'output' folder.

**watch**

  -Folder1
     -File1.mp4
     -File2.mp4
  -Folder2
    -Video1.mp4
    -Video2.mp4

**output**

File1.mp4
File2.mp4
Video1.mp4
Video2.mp4

Expected folder structire of output is same as 'watch' otherwise I have to manually segeregate all folders which is very time consuming.

**Comment by jlesage** [comment, marked as answer]
Did you set `AUTOMATED_CONVERSION_OUTPUT_SUBDIR` to `SAME_AS_SRC` ?
https://github.com/jlesage/docker-handbrake?tab=readme-ov-file#environment-variables

**Reply by techpara** [reply]
Thank you @jlesage. 

I think default value of AUTOMATED_CONVERSION_OUTPUT_SUBDIR should be SAME_AS_SRC
